lib/monitor/monitor.go

```diff
@@ -76,14 +73,12 @@ func NewMonitor(stats *Stats) *Monitor {
 // It needs the stats struct pointer to update when measures come
 // Return an error if something went wrong during the connection setup
 func (m *Monitor) Listen() error {
-	ln, err := net.Listen("tcp", Sink+":"+strconv.Itoa(m.SinkPort))
+	ln, err := net.Listen("tcp", Sink+":"+strconv.Itoa(DefaultSinkPort))
 	if err != nil {
```

Comment:
Why did you throw away the `m.SinkPort` here? We wanted to avoid a global variable and put it into the monitor-struct, which is more modular than having a global variable.